content/projects/data-science-specific/dashboards/_index.md

```diff
@@ -5,12 +5,10 @@ ready: True
 
```

Comment:
`noform:True` in the frontmatter. Otherwise it's pretty much guarenteed that the recruits will follow the wrong link.

Reply:
added noform:True